resources/styleheets/base/_login.scss

```diff
@@ -0,0 +1,214 @@
+body.login{
+    /* background: linear-gradient(to right, $background1, $background2); */
+    font-family: monospace;
+    background-image: url(resources/img/login_fondo.svg);
+    background-size: cover;
+    @include flex-center;
+}
+
+.contenido{
+    margin: 2.5rem;
+    height: 87vh;
+    width: 78%;
+    @include flex-center;
+    border-radius: 1rem;
+    color: $font;
+    border: 0.05rem solid $fondo_input;
+    box-shadow: $boxshadow2;
+    background-color: $primary-color;
+}
+
+.present{
+    height: 18rem;
+    order: 1;
+}        
+
+.left{
+    flex: 0.7;
+    padding: 2rem;
+    @include flex-grid;
+}
+
+.right{
+    flex: 0.5;
+    border-radius: 2rem;
+    @include flex-grid;
+
+    .inicio{
+        font-size: 2em;
+        @include flex-grid;
+    }
+
+    .logo{
+        @include flex-center;
+        font-size: 0.9em;
+        color: $font;
+        order: 3;
+    }
+
+}
```

Comment:
its not necessary the class .left and .right, because the tree looks like this
└── div.content //contenido
        └── div.right
        └── div.left
so can be made as
.content {
  div:first-child {
    // .left style goes here
  }
  
  div:last-child {
    // .right style goes here
  }
}
this is because the selectors will apply the ruleset to the first child div or the last child div, so the class can be eliminate making the html cleaner